### ISSUE TITLE
Issues with Running Robyn

### ISSUE THREAD

**Issue by DongHarry-Kang** [facebookexperimental/Robyn#844]
Hi Gurus -

I am a rookie in Robyn and R, and here are my two questions regarding new issues with Robyn. Could somebody such as @gufengzhou help me here? thank you very much! Please forgive my unprofessional words below! I actually think the below questions are the same problem so really appreciate your help

Problem 1:
System cannot recognize Python path in R: When I try installing nevergrad, after I used the code below
Sys.setenv(RETICULATE_PYTHON = "~/.virtualenvs/r-reticulate/bin/python")

I obtained the error message:
py_discover_config(required_module, use_environment) でエラー:  (In Japanese, エラー means error)
  Python specified in RETICULATE_PYTHON (~/.virtualenvs/r-reticulate/bin/python) does not exist
![image](https://github.com/facebookexperimental/Robyn/assets/146807827/77c997d8-b300-4fca-987f-1c3a62b72e8b)

Problem 2:
When I run Robyn, I got the error that Nevergrad is never installed.. with the below error message (In Japanese, エラー means error)

![image](https://github.com/facebookexperimental/Robyn/assets/146807827/0e270ce2-49d7-43a6-9682-85911048c553)

**Comment by DongHarry-Kang** [comment]
Hi Guru -

I am happy to report that I was able to resolve this issue. Somehow there is an issue on how folder was set up. I was able to resolve this issue by manually create a folder path for Python like below. I created bin/python folder under r-reticulate

![image](https://github.com/facebookexperimental/Robyn/assets/146807827/f5cb4e47-029a-4bc6-b812-a3c8d31a60ca)

As the result, when I run py_config(), I got the below result.

![image](https://github.com/facebookexperimental/Robyn/assets/146807827/f0e02b0a-3d05-4110-8e4f-e1b971457ef4)

And I can run Robyn again now
